apps/src/templates/certificates/GraduateToNextLevel.jsx

```diff
@@ -30,9 +33,9 @@ const GraduateToNextLevel = ({scriptName, courseTitle, courseDesc}) => {
 };
 
 GraduateToNextLevel.propTypes = {
-  scriptName: PropTypes.string.isRequired,
-  courseTitle: PropTypes.string.isRequired,
-  courseDesc: PropTypes.string.isRequired
+  scriptName: PropTypes.string,
+  courseTitle: PropTypes.string,
+  courseDesc: PropTypes.string
```

Comment:
For unit testing this component, I would suggest the following test cases:
1. scriptName in nextLevelCourseCards
2. scriptName not in nextLevelCourseCards
3. blank scriptName (or, make it required instead)